CIP-0000.md

```diff
@@ -0,0 +1,195 @@
+	* ➡️ Last Call — If agreeable, the CIP editor will assign Last Call status and set a review end date (review-period-end), normally 14 days later.
+	* ❌ Last Call — A request for Last Call status will be denied if material changes are still expected to be made to the draft. We hope that CIPs only enter Last Call once, so as to avoid unnecessary noise on the RSS feed.
+* **Last Call** — This CIP will be listed prominently on the  [https://cips.celo.org/](https://cips.celo.org/) website. **NOTE: CIP website is not live yet**
+	* ❌ — A Last Call which results in material changes or substantial unaddressed technical complaints will cause the CIP to revert to Draft.
+	* ➡️ Accepted (Ring 0-1 CIPs only) — A successful Last Call without material changes or unaddressed technical complaints will become Accepted.
+	* ✅ Final (Ring 0-1 or non-Standards CIPs) — A successful Last Call without material changes or unaddressed technical complaints will become Final.
+* **Accepted (Ring 0-1 CIPs only)** — This status signals that material changes are unlikely and Celo client developers should consider this CIP for inclusion. Their process for deciding whether to encode it into their clients as part of a hard fork is not part of the CIP process.
+	* ⬅️ Draft — The Core Devs can decide to move this CIP back to the Draft status at their discretion. E.g. a major, but correctable, flaw was found in the CIP.
+	* ❌ Rejected — The Core Devs can decide to mark this CIP as Rejected at their discretion. E.g. a major, but uncorrectable, flaw was found in the CIP.
+	* ✅ Final — Standards Track Ring 0-1 CIPs must be implemented in the Celo client before it can be considered Final. When the implementation is complete and adopted by the community, the status will be changed to “Final”.
+* **Final** — This CIP represents the current state-of-the-art. A Final CIP should only be updated to correct errata.
+
+Other exceptional statuses include:
+* **Active** — Some Informational and Process CIPs may also have a status of “Active” if they are never meant to be completed. E.g. CIP 0000 (this CIP).
+* **Abandoned** — This CIP is no longer pursued by the original authors or it may not be a (technically) preferred option anymore.
+	* ➡️ Draft — Authors or new champions wishing to pursue this CIP can ask for changing it to Draft status.
+* **Rejected** — A CIP that is fundamentally broken or a Ring 0-1 CIP that was rejected by the Core Devs and will not be implemented. A CIP cannot move on from this state.
+* **Superseded** — A CIP which was previously Final but is no longer considered state-of-the-art. Another CIP will be in Final status and reference the Superseded CIP. A CIP cannot move on from this state.
+
+## What belongs in a successful CIP?
```

Comment:
Can we move this into the template https://github.com/celo-org/celo-proposals/blob/master/CIPs/template.md ?

Reply:
I prefer keeping it here if that's ok so everyone has one place to understand the entire process. I will also be updating the template after this Draft is approved and merged because it needs to be expanded as well.

Reply:
Sorry yeah I should have not used the word move, but copy!

Reply:
Makes sense! Will add it as well to the template guide :)